chains/build_spec.js

```diff
@@ -0,0 +1,196 @@
+#!env node
```

Comment:
This is a script that builds `chains/{chain}/chain-spec.js` and `chains/{chain}/chain-spec-raw.js` based on data from `chains/{chain}/chain-config.json` and other sources (e.g. `compound-config` and `ethereum/networks/ropsten.json`)